routes/web.php

```diff
@@ -12,13 +12,15 @@
 */
   Route::get('/', function () {
       return view('welcome');
-  });
+  })->middleware('guest');
```

Comment:
just as a note to @SplitSet3R/box-end , this is a naked route this, we put the middleware check in the routes rather than in the construct (ie: it doesn't go through the controller to display it)

@brynbeaudry is it worthwhile to use a pagescontroller for naked routes to set the middleware check there?